src/EnergyPlus/SingleDuct.cc

```diff
@@ -2061,21 +2050,21 @@ namespace SingleDuct {
 		if ( Sys( SysNum ).ZoneMinAirFracMethod == FixedMin ) {
 			if ( ZoneSizingRunDone ) {
 				if ( CurZoneEqNum > 0 ) {
-					// if zone sizing values are available
-					if ( FinalZoneSizing( CurZoneEqNum ).DesCoolMinAirFlowFracUsInpFlg ) {
-						// if the user input a value for min flow frac in Sizing:Zone, use it
-						FixedMinAirDes = FinalZoneSizing( CurZoneEqNum ).DesCoolMinAirFlowFrac * Sys( SysNum ).MaxAirVolFlowRate;
-					}
-					else {
-						// otherwise use the combined defaults or other user inputs stored in DesCoolVolFlowMin
+					// use the combined defaults or other user inputs stored in DesCoolVolFlowMin
+					if ( Sys( SysNum ).MaxAirVolFlowRate > 0.0 ) {
 						FixedMinAirDes = FinalZoneSizing( CurZoneEqNum ).DesCoolVolFlowMin;
+					} else {
+						MinAirFlowFracDes = 0.0;
 					}
 				}
 			}
 			else {
-				// no zone sizing values available; use max of min frac = 0.2 and 0.000762 [m3/s-m2]
-				FixedMinAirDes = max( 0.2*Sys( SysNum ).MaxAirVolFlowRate, 0.000762 * Zone( ZoneNum ).FloorArea * Zone( ZoneNum ).Multiplier *
-					Zone( ZoneNum ).ListMultiplier );
+				// if no zone sizing values available; use max of min frac = 0.2 and 0.000762 [m3/s-m2]
+				if ( Sys( SysNum ).MaxAirVolFlowRate > 0.0 ) {
+					FixedMinAirDes = max( 0.2 * Sys( SysNum ).MaxAirVolFlowRate, 0.000762 * Zone( ZoneNum ).FloorArea * Zone( ZoneNum ).Multiplier * Zone( ZoneNum ).ListMultiplier );
+				} else {
+					MinAirFlowFracDes = 0.0;
+				}
```

Comment:
It's used [here](https://github.com/NREL/EnergyPlus/blob/6259aeca0d3a457e150606567b0197209c527193/src/EnergyPlus/SingleDuct.cc), as "always" but now it's more correct for situations where the required min flow is greater than the design flow based on load.